include/SongEditor.h

```diff
@@ -197,6 +205,13 @@ protected slots:
 
 	void updateSnapLabel();
 
+	void showZoomingSliderFloat();
```

Comment:
After some more testing I think the zoom float when scrolling is just annoying and not needed. We don't have that in PianoRoll or Audacity for example.

But I still think the slots should be in `SongEditor` next to their buddies.

Reply:
Ok. I agree. I remove zoomfloat for mouse scrolling.
But I don't get what you say about these slots. They are for the m_zoomingSlider slider that is in SongEditorWindow. Don't know what you mean but "their buddies"

Reply:
There are similar instances of `TextFloat` and slots to update them in `SongEditor` for the master pitch and volume. I suggest you place your slots and `TextFloat` there just to keep things organized.

(Alternatively you ditch the zoom float completely, since the numerical value doesn't really matter... What-you-see-is-what-you-get. But I don't know if other people would agree with me on that)

Reply:
Those sliders are "weird" because they don't belong to SongEditor but MainWindow. I don't know why they are there. Anyway, I think that we shouldn't mix views with controllers, so I think it's better to keep them on SongEditorView. Don't you think?

Reply:
You're absolutely right. Just because it was done wrong before doesn't mean we have to continue doing it wrong.